static/app/views/organizationStats/getReasonGroupName.ts

```diff
@@ -130,6 +131,8 @@ function getRateLimitedReasonGroupName(reason: RateLimitedReason | string): stri
   }
 
   switch (reason) {
+    case RateLimitedReason.ORG_QUOTA:
+      return 'org quota';
```

Comment:
@jan-auer what do we want to call it? I was not sure if we want to differentiate or display it as 'quota' as we do in line 129 https://github.com/getsentry/sentry/blob/3efa8cf306b9650d8129ffeaca77601b833b1009/static/app/views/organizationStats/getReasonGroupName.ts#L128-L130

Reply:
With the word "quota", in the product we usually refer to the monthly quota that the user pays for. It may be a bit inconsistent, that the word "quota" is used for this internal reason, but we should refer to it as a (rate) limit. 

I took another look at all rate limit reasons and would suggest the following:
- `endsWith('_usage_exceeded')` -> "quota"
- `endsWith('_disabled')` -> "disabled"
- `org_quota`, `project_quota` -> "global limit" (in docs explain that this is an old setting available to some orgs to configure rate limits on organization- and project level)
- `key_quota` -> "DSN limit" (better than "key limit")
- `spike_protection`, `smart_rate_limit` -> "spike protection"
- all other can remain "internal", these are abuse limits and when the organization is disabled.

Potentially we could think of a better description than "disabled"

Reply:
@jan-auer can you please give me a description for disabled? I will update the docs

Reply:
here is the [PR](https://github.com/getsentry/sentry-docs/pull/10947) for the docs